Northstar.Custom/mod/scripts/vscripts/gamemodes/sh_gamemode_gg.gnut

```diff
@@ -168,4 +140,39 @@ void function CreateGamemodeGG()
 array<GunGameWeapon> function GetGunGameWeapons()
 {
 	return file.weapons
+}
+
+string function Join( array<string> strings, string separator )
```

Comment:
Might be useful to move this to `Northstar.CustomServers/mod/scripts/vscripts/_utility.gnut` so other scripts can use it